.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: Weekly Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 00:00 on Monday
+    - cron: "0 0 * * 1"
+
+jobs:
+  bump-version-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch all history for all branches and tags
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+
+      - name: Install Dependencies
+        run: npm install
+
```

Comment:
```suggestion
```

I think this is unneeded, the test run just says that there is no `package.json`.